DC-144: Check health endpoint during cert import process

patrick-schweizer opened the pull request:
This PR contains as well:

- adopting to newest OIA interfaces (only returning metrics once)
- exposing the health check to Karaf: _opennms:health-check command_
- some restructuring

freddy61025 commented on plugin/src/main/java/org/opennms/plugins/cloud/tsaas/GrpcConnection.java:
```diff
@@ -0,0 +1,165 @@
+    private ManagedChannel managedChannel;
+    private TimeseriesGrpc.TimeseriesBlockingStub clientStub;
+
+    public void init(final TsaasConfig config, final SecureCredentialsVaultUtil scvUtil) {
```
Why not make it as a constructor? So that no one will miss calling init. Also you don't need to do the destroy in init.

freddy61025 replied:
Also I think the params better to do null check.

freddy61025 reviewed (approved):
LGTM just one minor question.

patrick-schweizer commented:
@freddy61025 I addressed your comments and also cleaned up the code a bit more. There is now only one way to import the certs (via auto import from etc file)
